src/shipraT/SquareCube3.java

```diff
@@ -0,0 +1,19 @@
+package shipraT;
+
+public class SquareCube3 {
+	public void squarecube() { //method creation
```

Comment:
Try to write generic method(defined parameter in method) which is useful for other as well